src/assets/pages/blog.json

```
@@ -1,5 +1,14 @@
+      "banner": "https://querido-diario-static.nyc3.cdn.digitaloceanspaces.com/blog/post0045/banner-post45.png",
+      "date": "05/15/2024",
```

Data precisa ser modificada para o dia da publicação do conteúdo
```suggestion
      "date": "10/01/2024",
```